Markdown/Code Block Syntaxes.sublime-completions

```diff
@@ -26,41 +56,107 @@
 			"details": "Specifies <code>C++</code> code highlighting"
 		},
 		{
-			"trigger": "clj",
-			"annotation": "Clojure",
+			"trigger": "h",
+			"annotation": "C header",
 			"kind": ["markup", "s", "Syntax"],
-			"details": "Specifies <code>Clojure</code> code highlighting"
+			"details": "Specifies <code>C</code> code highlighting"
 		},
 		{
-			"trigger": "clojure",
-			"annotation": "Clojure",
+			"trigger": "hpp",
+			"annotation": "C++ header",
 			"kind": ["markup", "s", "Syntax"],
-			"details": "Specifies <code>Clojure</code> code highlighting"
+			"details": "Specifies <code>C++</code> code highlighting"
 		},
 		{
-			"trigger": "cpp",
-			"annotation": "C++",
+			"trigger": "hxx",
+			"annotation": "C++ header",
+			"kind": ["markup", "s", "Syntax"],
+			"details": "Specifies <code>C++</code> code highlighting"
+		},
+		{
+			"trigger": "h++",
+			"annotation": "C++ header",
 			"kind": ["markup", "s", "Syntax"],
 			"details": "Specifies <code>C++</code> code highlighting"
 		},
+
+		// C#
+		{
+			"trigger": "csharp",
+			"annotation": "C#",
+			"kind": ["markup", "s", "Syntax"],
+			"details": "Specifies <code>C#</code> code highlighting"
+		},
 		{
 			"trigger": "cs",
 			"annotation": "C#",
 			"kind": ["markup", "s", "Syntax"],
 			"details": "Specifies <code>C#</code> code highlighting"
 		},
 		{
-			"trigger": "csharp",
+			"trigger": "c#",
```

Comment:
I have not tested this, but I'd assume this won't work as a completion.

Reply:
So you may want to give it a shot?